lib/facebook/messenger/bot.rb

```diff
@@ -171,6 +171,15 @@ def default_options
             }
           )
         end
+
+        #
+        # Returns the graph api version defined in the provider.
+        #
+        # @return [String] graph api version.
+        #
+        def graph_api_version
+          Facebook::Messenger.config.provider.graph_api_version
+        end
```

Comment:
Each resource has one method like this, to the eventuality that we need to lock a resource to a specific version.